siuba/tests/test_sql_misc.py

```diff
@@ -38,7 +38,8 @@ def test_raw_sql_mutate_refer_previous_raise_dberror(backend, skip_backend, df):
     if backend.name == "duckdb":
```

Comment:
With a new enough duckdb-engine version, this if clause shouldn't be required at all?

Reply:
Ah I see, that's unfortunate